### DISCUSSION TITLE
上传文件返回413状态码

### DISCUSSION THREAD

**Discussion by asheswang** [apache/dolphinscheduler#6413] [Q&A]
dolphinscheduler版本是1.3.6，使用资源中心上传文件到HDFS的时候返回状态码413，基本只能传30M左右以下的文件。
看网上很多都说是修改nginx配置，但照官网文档进行配置，1.3.x版本无需进行nginx配置，因此我的机器上并没有安装nginx。
请问这个要怎么解决？

**Comment by davidzollo** [comment]
yes, 1.3 version doesn't need to use nginx
you can check the following items in the `conf/application-api.properties`.
```
spring.servlet.multipart.max-file-size=1024MB
spring.servlet.multipart.max-request-size=1024MB
```
you can also search some answers for this question,  DolphinScheduler use Springboot as web service frame

**Reply by asheswang** [reply]
> yes, 1.3 version doesn't need to use nginx you can check the following items in the `conf/application-api.properties`.
> 
> ```
> spring.servlet.multipart.max-file-size=1024MB
> spring.servlet.multipart.max-request-size=1024MB
> ```
> 
> you can also search some answers for this question, DolphinScheduler use Springboot as web service frame

I have checked and it 's correct. So I don 't kown how to solve the problem.
